src/AspNetCore/Inc/applicationinfo.h

```diff
@@ -143,6 +143,19 @@ class APPLICATION_INFO
         return m_pfnAspNetCoreCreateRequestHandler;
     }
 
+    VOID
+    ShutDown(
+        VOID
+    );
+
+    static
+    VOID
+    ShutDownWrapper(APPLICATION_INFO* pApplicationInfo, PVOID context)
```

Comment:
what's the context reserved for?

Reply:
It isn't used at all, it is called from the HASH_TABLE.